### PR TITLE
[ISSUE #2370]【Code Clean】Remove warning codes in [shenyu-plugin-grpc] module

### PR DESCRIPTION
**Pull request by li-keguo**
about issue no: #2370

work list:

- [x] Remove warning codes in [shenyu-plugin-grpc] module.
- [x] Remove warning codes in [shenyu-plugin-base] module.

### CODE REVIEW

**Review by songyuequan — COMMENTED** [review]
thank you for your help to  clean some code in plugin-base

File: shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/AbstractShenyuPlugin.java [hunk 1]
```diff
@@ -128,13 +128,13 @@ private Boolean filterRule(final RuleData ruleData, final ServerWebExchange exch
     }
 
     private void selectorLog(final SelectorData selectorData, final String pluginName) {
-        if (selectorData.getLogged()) {
+        if (Boolean.TRUE.equals(selectorData.getLogged())) {
```
Comment (yu199195) [inline shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/AbstractShenyuPlugin.java:131]: why modify this？

File: shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/utils/ResponseUtils.java [hunk 2]
```diff
@@ -80,20 +84,21 @@ public static ClientResponse buildClientResponse(final ServerHttpResponse respon
...
      * release source.
+     *
      * @param outputMessage CachedBodyOutputMessage
-     * @param throwable Throwable
+     * @param throwable     Throwable
      * @return Mono.
      */
     public static Mono<Void> release(final CachedBodyOutputMessage outputMessage, final Throwable throwable) {
-        if (outputMessage.getCache()) {
+        if (Boolean.TRUE.equals(outputMessage.getCache())) {
```
Comment (yu199195) [inline shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/utils/ResponseUtils.java:96]: why modify this？

File: shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/AbstractShenyuPlugin.java (hunk 1 above)
Comment (li-keguo) [reply shenyu-plugin/shenyu-plugin-base/src/main/java/org/apache/shenyu/plugin/base/AbstractShenyuPlugin.java:131]: `Boolean logged` maybe is null , if null ,thow NullPointException

**Review by yu199195 — APPROVED** [review]

**Pull request merged**